kitchen.vagrant.yml

```diff
@@ -27,14 +29,31 @@ platforms:
   - name: ubuntu-18.04
 
 suites:
-  - name: default
-    run_list:
-      - recipe[os_prepare]
-  - name: find_files
-    run_list:
-      - recipe[os_prepare::find_files]
-    verifier:
-      inspec_tests:
-        - test/kitchen/policies/find_files
-    excludes:
-      - freebsd-12
+- name: resources-core
+  run_list:
+  - recipe[build-essential]
+  - recipe[os_prepare]
+  - recipe[audit]
+  verifier:
+    name: inspec
+    inspec_tests:
+      - test/kitchen/policies/find_files
+  attributes:
+    audit:
+      attributes:
+        audit_attribute: 'Attribute Override!'
+      insecure: true
+      reporter: ['json-file','chef-automate']
+      fetcher: 'chef-automate'
+      json_file:
+        location: /tmp/json_export.json
+      profiles:
+        - name: integration
+          url: https://github.com/inspec/inspec-integration-profile/archive/master.zip
```

Comment:
This section looks a lot like the Kitchen file for the audit cookbook, which is complicated in order to exercise lots of its features. I hope this can be simplified in the future.

Reply:
Yes! I kinda lifted our old config a bit just to get kitchen.vagrant working again for debugging train#458 with @zenspider. I have another update I'm going to push that drops the audit bits, but i'll do that in another PR along with a few actual 'resource-core' tests.